README.md

```diff
@@ -39,6 +39,12 @@ See https://pekko.apache.org for the documentation including the API docs. The d
      - the `index.html` file will appear in `target/paradox/site/main/`
 - `sbt unidoc` will build the Javadocs for all the modules and load them to one place (may require Graphviz, see Prerequisites above)
      - the `index.html` file will appear in `target/scala-2.13/unidoc/`
+- `sbt sourceDistGenerate` will generate source release to `target/dist/`
+- The version number that appears in filenames and docs is derived, by default. The derived version contains the most git commit id or the date/time (if the directory is not under got control). 
```

Comment:
type here, `under got control` should be `under git control`

Reply:
thanks - fixed the typo